skietbaan_fe/src/components/Login.js

```diff
@@ -51,7 +51,7 @@ class App extends Component {
         "Email": this.state.emailValue,
         "Password": hash,
       }
-      fetch("http://skietbaan.retrotest.co.za/api/User", {
+      fetch("https://api.skietbaan.retrotest.co.za/api/User", {
```

Comment:
I am sure you guys have global that js where you store this please use that variable

Reply:
We actually dont have one

Reply:
@RetroGuillaume check your Ryver messages